x-pack/solutions/security/plugins/security_solution/server/endpoint/services/actions/clients/microsoft/defender/endpoint/ms_defender_endpoint_actions_client.ts

```diff
@@ -95,6 +98,11 @@ const MDE_ACTION_FETCH_RETRY_CONFIG = {
   factor: 1.5,
 };
 
+// max size of script output file to retrieve
+// little more than endpoint execute stderr/stdout threshold of 2kB
+// to account for mde output file containing extra metadata
+const RUNSCRIPT_OUTPUT_FILE_MAX_SIZE_BYTES = 2.5 * 1024; // 2.5 KB
```

Comment:
2kb seems small to me. Is this what the Endpoint team uses when they build the response for Endpoint execute?

Reply:
Yes. Endpoint uses `2kb` for each `stdout` and `stderr` for limiting trailing end sizes. cc @intxgo

Reply:
I am now seeing some cases where both stdout and stderror are present so I'm going to update this to 4.5 kB